chromium/background.js

```diff
@@ -384,7 +386,7 @@ function onBeforeRequest(details) {
     // failing.
     if (shouldCancel) {
       if (!newuristr) {
-        return {redirectUrl: canonical_url.replace(/^http:/, "https:")};
+        return {redirectUrl: details.url.replace(/^http:/, "https:")};
```

Comment:
IMHO, this is change is not necessary, better not to change the existing behavior.

Reply:
This is actually why they are replaced with null but not replaced back.

Reply:
@cschanaj Even better now, bug is fixed but the existing behavior is kept.